### DISCUSSION TITLE
No log line matching the '' filter on log

### DISCUSSION THREAD

**Discussion by nitrup** [OlivierGaland/CashFactory#17] [Q&A] [answered]
Hi all,

I used Unbuntu 20.04 LTS

On the container bellow I have the error on log : No log line matching the '' filter on log

[cashfactory_Earnapp_1]

I follow the installation description all is running but on log this error normal ?

Thanks

**Comment by OlivierGaland** [comment, marked as answer]
Hello,

This is normal for earnapp, I have same behaviour on my stack, best way to check if everything is ok is to log on dashboard and check that the devices are online (don't forget to declare them -there is a special procedure for earnapp in the main readme-).

Olivier.
